integration/test-integration/src/test/resources/openAPIs/operation_level_ratelimit_openAPI.yaml

```diff
@@ -0,0 +1,114 @@
+swagger: '2.0'
+info:
+  description: 'This is a sample server Petstore server.  You can find out more about Swagger at [http://swagger.io](http://swagger.io) or on [irc.freenode.net, #swagger](http://swagger.io/irc/).  For this sample, you can use the api key `special-key` to test the authorization filters.'
+  version: 1.0.5
+  title: Swagger Petstore
```

Comment:
Related to https://github.com/wso2/product-microgateway/pull/3142/files#r1049777998